backend/main.py

```
@@ -11,208 +17,180 @@
-                    "cup_to_disc_ratio": cup_to_disc_ratio,
-                    "image_base64": img_base64
-                }
+                "status": "success", "message": "Analysis completed!",
+                "data": {"has_glaucoma": is_glaucoma, "cdr": cup_to_disc_ratio, "image_base64": img_base64, "saved_to_db": bool(current_user)}
             }) + "\n"
 
-            if os.path.exists(tmp_path):
-                os.remove(tmp_path)
-
+            if os.path.exists(tmp_path): os.remove(tmp_path)
         except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)